Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml

```diff
@@ -2,7 +2,7 @@
   name: pickaxe
   parent: BaseItem
   id: Pickaxe
-  description: Notched to perfection, for jebbing it into rocks
+  description: Notched to perfection, for jamming it into rocks
```

Comment:
```suggestion
  description: Notched to perfection, for jabbing it into rocks.
```

Reply:
Jeb was also a minecraft dev, so I assume that "jebbing" was meant to be an intentional reference

Reply:
Now that I think of it, "jamming" might be better, as the word "jabbing" is more for spears and the use thereof.